### PR TITLE
secman

### PR DESCRIPTION
**Pull request by abdfnx**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/secman.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  url "https://cli-files.secman.dev/secman-v6.0.71/secman-v6.0.71.tar.gz"
+  sha256 "2a9c03df9b444e9c110b640c2e22ef4db147623cb728f7b2c68f2bc488d0e230"
+  license "MIT"
+
+  livecheck do
+    url :homepage
+    regex(/href=.*?aamath[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  depends_on "scmn-dev/secman/sm-node"
```
Comment (carlocab) [inline Formula/secman.rb:13]: Formulae in Homebrew/core should not have dependencies in external taps.

File: Formula/secman.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  def install
+    inreplace "bin/secman", /^CLIENT_HOME=/, "export SECMAN_OCLIF_CLIENT_HOME=#{lib/"client"}\nCLIENT_HOME="
+    inreplace "bin/secman", "\"$DIR/node\"", Formula["sm-node"].opt_bin/"node"
+    libexec.install Dir["*"]
+    bin.install_symlink libexec/"bin/secman"
+    ENV["SM_PROVIDER"] = "brew"
```
Comment (carlocab) [inline Formula/secman.rb:20]: This line doesn't do anything.

File: Formula/secman.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  end
+
+  test do
+    system bin/"secman", "-v"
```
Comment (carlocab) [inline Formula/secman.rb:24]: Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

**Review by carlocab — CHANGES_REQUESTED** [review]
Thanks, @abdfnx. A few comments.

**Comment by carlocab** [comment]
Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, use the following guidelines to make sure that @BrewTestBot can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

**Comment by abdfnx** [comment]
thank you @carlocab

File: Formula/secman.rb (hunk 2 above)
Comment (abdfnx) [reply Formula/secman.rb:20]: do you mean `ENV["SM_PROVIDER"] = "brew"` ? @carlocab

Comment (carlocab) [reply Formula/secman.rb:20]: Yes. You're setting an env variable for the build environment, but nothing else occurs after that to use the variable you've set.

**Pull request closed without merging**